# Foxk1_manuscript

## Main Figures

### Figure 2
|  | Figures | Link | 
| --- | --- | --- | 
| GO pathway analysis highlights pathways and development related terms in D3 control EBs, the x-axis represents the counts of genes in each GO term. | Fig. 2a | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/RNAseq_Figure_2_Supplementary_3.ipynb) |
| The heatmap represents  upregulated and downregulated in the control EBs vs. the Foxk1 KO EBs at D3 | Fig. 2b | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/RNAseq_Figure_2_Supplementary_3.ipynb) |
| GO pathway analysis highlights pathways and development related terms in D5 control EBs, the x-axis represents the counts of genes in each GO term. | Fig. 2c | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/RNAseq_Figure_2_Supplementary_3.ipynb) |
| The heatmap represents  upregulated and downregulated in the control EBs vs. the Foxk1 KO EBs at D5 | Fig. 2d | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/RNAseq_Figure_2_Supplementary_3.ipynb) |


### Figure 3
|  | Figures | Link | 
| --- | --- | --- | 
| Enriched heatmap of the nucleoATAC data showing more nucleosome free regions (NFR) in control samples as compared to Foxk1 KO samples at D3 at the Foxk1 binding sites. | Fig. 3a | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/New_Figure_3ab_FOXK1_Nucleoatac_KO_ND_enriched_heatmap_showing_NFR_and_NOR.ipynb) |
| Enriched heatmap of the nucleoATAC data showing more nucleosome free regions (NFR) in control samples as compared to Foxk1 KO samples at D5 at the Foxk1 binding sites. | Fig. 3b | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/New_Figure_3ab_FOXK1_Nucleoatac_KO_ND_enriched_heatmap_showing_NFR_and_NOR.ipynb) |
| Venn Diagram shows the overlap of increased accessibility and upregulated genes between ATACseq and RNAseq, respectively, in the control group at D3 over the Foxk1 KO group | Fig. 3c | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/Figure_3cdef_Supplementary_4_RNA_ATAC_Integration_chromVAR.ipynb) |
| Venn Diagram shows the overlap of increased accessibility and upregulated genes between ATACseq and RNAseq, respectively, in the control group at D5 over the Foxk1 KO group | Fig. 3d | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/Figure_3cdef_Supplementary_4_RNA_ATAC_Integration_chromVAR.ipynb) |
| The heatmap shows commonly expressed transcription factors in both the ATACseq and RNAseq datasets upregulated and downregulated in the control EBs over the Foxk1 KO EBs at D3. | Fig. 3e | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/Figure_3cdef_Supplementary_4_RNA_ATAC_Integration_chromVAR.ipynb) |
| The heatmap shows commonly expressed transcription factors in both the ATACseq and RNAseq datasets upregulated and downregulated in the control EBs over the Foxk1 KO EBs at D5. | Fig. 3f | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/Figure_3cdef_Supplementary_4_RNA_ATAC_Integration_chromVAR.ipynb) |

### Figure 4
|  | Figures | Link | 
| --- | --- | --- | 
| Pathway analysis highlights GO pathways and development related terms in D10 control EBs, the x-axis represents the counts of genes in each GO term. | Fig. 4e | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/Figure_4ef_RNAseq_Foxk1_D10_analysis.ipynb) |
| The heatmap shows significantly (adjusted p-value < 0.05) differentially expressed transcripts upregulated and downregulated with a two fold between control EBs vs. Foxk1 null EBs at D10. | Fig. 4f | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/Figure_4ef_RNAseq_Foxk1_D10_analysis.ipynb) |
| The heatmap represents a significant (adjusted p-value < 1e-04) change in chromatin accessibility at D10 for control and Foxk1 KO. | Fig. 4g | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/Figure_4g_Day_10_FOXK1_chromVAR_ATAC_analysis.ipynb) |

### Figure 5
|  | Figures | Link | 
| --- | --- | --- | 
| RNAseq pathway analysis of Wnt signaling in D5 EBs comparing control and Foxk1 KO groups. | Fig. 5a | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/Figure_6a_and_5a_WNT_at_D5.ipynb) |
| The PCA of RNAseq of EB differentiation at D10 shows samples with similar gene expression cluster together. | Fig. 5e | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/RNAseq_Foxk1_D10_analysis_Control_and_Iwr1_KO_similarity.ipynb) |
| Venn diagram shows 320 genes up-regulated in both Ct and KO + IWR1 samples when compared to Foxk1 KO samples | Fig. 5f | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/RNAseq_Foxk1_D10_analysis_Control_and_Iwr1_KO_similarity.ipynb) |
|  Top 10 GO pathways and development terms significantly enriched using the genes commonly upregulated in control and Foxk1 KO + IWR1 conditions. | Fig. 5g | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/RNAseq_Foxk1_D10_analysis_Control_and_Iwr1_KO_similarity.ipynb) |
|The heatmap shows top 20 genes from two GO terms (muscle tissue development and striated muscle tissue development) commonly upregulated in control and Foxk1 KO + IWR1 samples. | Fig. 5h | [R](https://colab.research.google.com/github/djglab/Foxk1_manuscript/blob/main/notebooks/RNAseq_Foxk1_D10_analysis_Control_and_Iwr1_KO_similarity.ipynb) |
